.travis.yml

```diff
@@ -0,0 +1,56 @@
+sudo: false
```

Comment:
Removing `sudo: false` worked  as expected after removing reference to tests.sh.